packages/ui/src/components/Button/index.tsx

```diff
@@ -20,17 +22,21 @@ export function Button({
   className = '',
   children,
   onClick,
-  size,
+  size = '',
+  full = false,
+  icon,
 }: ButtonProps) {
+  const block = full ? 'block' : '';
   return (
     <ButtonStyles>
       <button
         type={submit ? 'submit' : 'button'}
         disabled={disabled}
         onClick={onClick}
-        className={`${className} ${type} ${size}`}
+        className={`${className} ${type} ${size} ${block}`}
         style={style}
       >
+        <span>{icon}</span>
```

Comment:
```suggestion
        { icon && <span>{icon}</span> }
```

Reply:
Right! So we don't have an empty `span` for no reason! I missed it!!! Shit. Thanks, thanks!